manifests/local-storage.yaml

```diff
@@ -82,6 +82,18 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        resources:
```

Comment:
If you change these manifests, you need to run `make download && make generate`, and check in the resulting bindata.

Reply:
I'm unsure of how to verify the generated bindata is actually correct. Running the zz_generated_bindata.go go code doesn't seem to work....

Reply:
It's autogenerated, so I wouldn't worry about it too much - but if you want to test it completely you can build k3s and then start up the resulting docker image as a test, and confirm that the resulting manifests dropped into /var/lib/rancher/k3s/server/manifests have the content you're expecting.

Reply:
Okay, thank you. I've committed the changes.